util.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+# -*- indent-tabs-mode: nil  -*-
```

Comment:
please keep the bash script header. what is this new change ``-*-``?

Reply:
It doesn't make sense to execute this script on its own. So the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) is not necessary here and removing it, ensures that the script cannot be executed on its own.

Reply:
> what is this new change -*-?

That's a hint to emacs how to configure indentation. You said, you prefer to follow Google's code-style and avoid tabs. My emacs' default preference is to use tabs...